modules/reindex/src/main/java/org/opensearch/index/reindex/remote/RemoteScrollableHitSource.java

```diff
@@ -186,8 +186,9 @@ public void onSuccess(org.opensearch.client.Response response) {
                             }
                             if (xContentType == null) {
                                 try {
+                                    logger.debug("Response didn't include Content-Type: " + bodyMessage(response.getEntity()));
                                     throw new OpenSearchException(
-                                        "Response didn't include Content-Type: " + bodyMessage(response.getEntity()));
+                                        "Response didn't include supported Content-Type, remote is likely not an OpenSearch instance");
                                 } catch (IOException e) {
```

Comment:
This is no longer relevant as call to `bodyMessage` is removed?

Reply:
The call to `bodyMessage` is still used to log the message.